lib/index.js

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2016-2022 Digital Bazaar, Inc. All rights reserved.
  */
 import * as bedrock from '@bedrock/core';
+import {existsSync} from 'node:fs';
```

Comment:
Sync? So sad... :(

Reply:
It matters little in this case.  But if you can decipher the docs related to async exists with stat/access... please provide a patch!